app/models/payment_providers/stripe_provider.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module PaymentProviders
+  class StripeProvider < BaseProvider
```

Comment:
just a naming thing, could it be better to have `PaymentProviders::Stripe` only?

Reply:
yeah I was thinking about this too, fair enough, you win!